src/Block/AuditBlockService.php

```diff
@@ -39,16 +39,21 @@ public function __construct(Environment $twig, AuditReader $auditReader)
 
     public function execute(BlockContextInterface $blockContext, ?Response $response = null): Response
     {
+        $template = $blockContext->getTemplate();
```

Comment:
can a block return null here?

Reply:
I assume that no because of 
```
public function configureSettings(OptionsResolver $resolver): void
    {
        $resolver->setDefaults([
            'limit' => 10,
            'template' => '@SonataDoctrineORMAdmin/Block/block_audit.html.twig',
        ]);
    }
```